diff --git a/images/custom/Containerfile b/images/custom/Containerfile
--- a/images/custom/Containerfile
+++ b/images/custom/Containerfile
@@ -85,5 +85,8 @@ RUN useradd -ms /bin/bash frappe \
     && chmod 755 /usr/local/bin/nginx-entrypoint.sh \
     && chmod 644 /templates/nginx/frappe.conf.template
 
+ARG NODE_OPTIONS=--max-old-space-size=4096
+ENV NODE_OPTIONS=${NODE_OPTIONS}
+
 
 FROM base AS builder
